src/Voron/Impl/Transaction.cs

```diff
@@ -482,5 +497,82 @@ public static void DebugDisposeReaderAfterTransaction(Transaction tx, BlittableJ
             // helper that is used across the board, but it is meant to assert stuff during debug only
             tx.LowLevelTransaction.OnDispose += state => reader.Dispose();
         }
+
+        public void DeleteTable(string name)
+        {
+            var tableTree = ReadTree(name, RootObjectType.Table);
+            
+            var writtenSchemaData = tableTree.DirectRead(TableSchema.SchemasSlice);
+            var writtenSchemaDataSize = tableTree.GetDataSize(TableSchema.SchemasSlice);
+            var schema = TableSchema.ReadFrom(Allocator, writtenSchemaData, writtenSchemaDataSize);
+
+            var table = OpenTable(schema, name);
+
+            // delete table data
+
+            table.DeleteByPrimaryKey(Slices.BeforeAllKeys, x =>
+            {
+                bool isOwned = table.IsOwned(x.Reader.Id);
+                return isOwned;
```

Comment:
How can this _not_ be owned by this table if this is coming from this table?

Reply:
If the PK is global then we might get Id which isn't in the table we're deleting.

Reply:
Added: https://github.com/aviv86/ravendb/pull/1/commits/c18d645d2795fd655fb1372600234a2a8cc88323

Reply:
Okay, make sense